src/Uno.UI/UI/Xaml/Shapes/DefsSvgElement.wasm.cs

```diff
@@ -2,9 +2,9 @@
 using System.Collections.Generic;
 using System.Text;
 using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Wasm;
+using static Windows.UI.Xaml.Shapes.Shape;
```

Comment:
why this static using?

Reply:
Indeed, don't use static usings for this, this gets confusing, we're not at that level of character economy ;)